internal/controller/helmrelease_controller.go

```diff
@@ -643,6 +650,9 @@ func (r *HelmReleaseReconciler) adoptLegacyRelease(ctx context.Context, getter g
 	// Erase the last release revision from the status.
 	obj.Status.LastReleaseRevision = 0
 
+	// Set the last attempted post-renderers digest.
+	obj.Status.LastAttemptedPostRenderersDigest = postrender.Digest(digest.Canonical, obj.Spec.PostRenderers).String()
```

Comment:
Should we set this if there are no post renderers?

Reply:
No, It should be done only if present. Updated